lib/pages/get_wallet_page.dart

```diff
@@ -52,80 +51,80 @@ class GetWalletPage extends StatelessWidget {
       }
     }
 
-    return WalletConnectModalNavBar(
-      title: const WalletConnectModalNavbarTitle(
-        title: 'Get a wallet',
-      ),
-      child: Column(
-        children: [
-          Column(
-            mainAxisSize: MainAxisSize.min,
-            mainAxisAlignment: MainAxisAlignment.start,
-            children: walletWidgets,
-          ),
-          Divider(
-            height: 0,
-            thickness: 1,
-            // indent: 20,
-            // endIndent: 20,
-            color: themeData.colors.background300,
-          ),
-          const SizedBox(height: 8.0),
-          Text(
-            "Not what you're looking for?",
-            textAlign: TextAlign.center,
-            style: TextStyle(
-              fontSize: 16.0,
-              color: themeData.colors.foreground100,
+    return Web3ModalNavbar(
+      title: 'Get a wallet',
+      child: SafeArea(
+        child: Column(
+          children: [
+            Column(
+              mainAxisSize: MainAxisSize.min,
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: walletWidgets,
             ),
-          ),
-          const SizedBox(height: 4.0),
-          Padding(
-            padding: const EdgeInsets.symmetric(
-              horizontal: 40,
+            Divider(
+              height: 0,
+              thickness: 1,
+              // indent: 20,
```

Comment:
Are these needed?

Reply:
This screen is not yet migrated to v3, I just added the NavBar on top of it but everything else is old. I'll be migrating these screens this week hopefully.